apps/src/containedLevels.js

```diff
@@ -61,11 +61,15 @@ export function postContainedLevelAttempt({
   attempts,
   onAttempt
 }) {
-  if (!hasContainedLevels || attempts !== 1) {
+  if (!hasContainedLevels) {
     return;
   }
-
   const isTeacher = getStore().getState().currentUser?.userType === 'teacher';
+
+  if (!isTeacher && attempts !== 1) {
```

Comment:
For my own understanding- Is it the same use case for students who have attempted 0 times _or_ 2+?

Reply:
Yes! I just had to write out the table of possible logical combinations to convince myself again though. Do you have any suggestions to make this clearer?